app/web/template/layout/page.gohtml

```diff
@@ -14,6 +17,7 @@
       <link rel="apple-touch-icon" sizes="152x152" href="/static/assets/images/govuk-apple-touch-icon-152x152.png" >
       <link rel="apple-touch-icon" href="/static/assets/images/govuk-apple-touch-icon.png" >
       <link href="/static/stylesheets/all.css" rel="stylesheet">
+      <script async src="/static/javascript/main.js"></script>
```

Comment:
I think we'll need to add hashes to these before caching them, like we have for lpa-frontend https://github.com/ministryofjustice/opg-sirius-lpa-frontend/blob/main/main.go#L69

Reply:
Have added the hashing in and working okay, but not 100% on my implementation